package.json

```diff
@@ -11,7 +11,8 @@
     "develop": "webpack-dev-server --config webpack.config.develop.js --open",
     "lint": "eslint *.js",
     "gh-pages": "rimraf ./docs && yarn build && cp -rv ./build/ ./docs",
-    "serve": "yarn build && cd build && http-server -o"
+    "serve": "yarn build && cd build && http-server -o",
+    "test": "karma start"
```

Comment:
oh!  let's add `rimraf ./reports` here before `karma start` to clean up the reports directory for each run and then this is good to go!